modules/hostap/Kconfig

```diff
@@ -198,6 +198,7 @@ config WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
 	select MBEDTLS_SERVER_NAME_INDICATION if MBEDTLS_BUILTIN
 	select MBEDTLS_X509_CRL_PARSE_C
 	select MBEDTLS_TLS_VERSION_1_2
+	select INSECURE
```

Comment:
I think, if we select INSECURE, the help for that option should contain some text explaining why it is not secure.